### DISCUSSION TITLE
No Python 3.9.x in Arch Linux

### DISCUSSION THREAD

**Discussion by tomlawesome** [UltimaHoarder/UltimaScraper#646] [General]
Hey, 

Is there a way to get a working older version that ran with 3.8.x? Arch/Manjaro don't currently support 3.9.x so since the upgrade to 3.9.x usage for the script, I can't use it anymore which sucks, as it's awesome. 

My only work around has been to run the script on my dual boot with Winblows, but that means I literally have to sit and wait for stuff to download which isn't super exciting. 

I appreciate this is a niche/temporary issue once Arch/Manjaro release Python 3.9.x into the wild, but in the meantime any help is appreciated.

Cheers.

**Comment by tomlawesome** [comment]
Hah... perhaps the Gods are with me? 

Finally decided to write a post and then Manjaro go ahead and release a major update.

**Reply by UltimaHoarder** [reply]
Sorry, I didn't see this earlier because I was busy, but the script is using 3.9 syntaxes so I can't roll back.

I'm glad they updated their OS so you can use it lmao.
